src/quo2/components/password/tips/view.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.password.tips.view
+  (:require [react-native.core :as rn]
+            [quo2.components.password.tips.style :as style]
+            [quo2.components.markdown.text :as text]))
+
+(defn view
+  "Options
+   - `text` Password tip string
+   - `completed?` Password tip state"
+  [{:keys [completed? text]}]
```

Comment:
nit: should text be just it's own param? We use this approach in a lot of components. right @Parveshdhull

Reply:
Are you suggesting to do `(defn view [text completed?])`  instead of using `:keys`?

Reply:
actually the other way, so the text is the second param and the first param is the options object.
```
(defn view [{:keys [completed?]} text]) 
```

Reply:
Is there any reason for preferring this pattern?

Reply:
One issue I see with this approach is that we always have to pass the options objects,

With the previous approach we can create a component by just passing `text` - `[quo/tips {:text "Numbers"}]`. But if we want to do the same with the new approach we always have to pass an object as the first param `[quo/tips {} "Numbers"]` (this usage may not useful for this component, but I'm curious why we prefer this pattern)

Reply:
The reason it's preferable (as far as I know) is because it is consistent across our other components. Additionally it allows us to pass through more complex text combinations and not just a string.

To avoid always passing an empty object you can do.
```
(defn view
  "Options
   - `completed?` Password tip state
   
   `text` Password tip string
   "
  ([{:keys [completed?]} text]
  [rn/view
   {:style               style/container
    :accessibility-label :password-tips}
   [text/text
    {:style  (style/tip-text completed?)
     :weight :regular
     :size   :paragraph-2} text]
   (when completed?
     [rn/view
      {:style               style/strike-through
       :accessibility-label :password-tips-completed}])])
([text] ( view {} text))
)
       
  ```

Reply:
another example which uses mutliple arity (not sure if that's the correct name) is the `custom-color` method in colors file:
https://github.com/status-im/status-mobile/blob/develop/src/quo2/foundations/colors.cljs#L216-L228

Reply:
Okay, got it. Thanks for the explanation.